index.html

```diff
@@ -44,8 +45,199 @@ <h6><a href="">LET'S CONNECT</a></h6>
+                <ul class = "skills-text">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <div class ="button-container">
+                <button class = "button1">See Project</button>
+                </div>
+            </div>
+
+        </section>
+        
+        <section class = "work-area-card">
+            <figure>
+                <img src="Images/Tonic-2.png" alt="tonic-one" class = "tonic-image">
+            </figure>
+            <div class = "work-text-card">
+                <header class = "work-area-header">Tonic</header>
+                <div class = "specification">
+                    <span class = "user-name">CANOPY</span>
+                    <img src = "Images/dot-symbol.svg" alt = "dot-symbol"/>
+                    <span class = "user-experience">Back End Dev</span>
+                    <img src = "Images/dot-symbol.svg" alt = "dot-symbol"/>
+                    <span class = "user-experience">2015</span>
+
+                </div>
+                <p class = "description">A daily selection of privately personalized reads. no accounts
+                    or sign-ups required.
+                </p>
+                <ul class = "skills-text">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <div class ="button-container">
+                <button class = "button1">See Project</button>
+                </div>
+            </div>
+
+        </section>
+        
+        <section class = "work-area-card">
+            <figure>
+                <img src="Images/stories-2.png" alt="tonic-one" class = "tonic-image">
+            </figure>
+            <div class = "work-text-card">
+                <header class = "work-area-header">Multi-Post <br>
+                    Stories</header>
+                <div class = "specification">
+                    <span class = "user-name">CANOPY</span>
+                    <img src = "Images/dot-symbol.svg" alt = "dot-symbol"/>
+                    <span class = "user-experience">Back End Dev</span>
+                    <img src = "Images/dot-symbol.svg" alt = "dot-symbol"/>
+                    <span class = "user-experience">2015</span>
+
+                </div>
+                <p class = "description">A daily selection of privately personalized reads. no accounts
+                    or sign-ups required.
+                </p>
+                <ul class = "skills-text">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <div class ="button-container">
+                <button class = "button1">See Project</button>
+                </div>
+            </div>
+
+        </section>
+    </article>
+
+    <!--About my section-->
+    
+    <article class = "about-myself-background">
```

Comment:
- [ ] As mentioned above, kindly make sure that your `About` section uses a `<section>` tag is it is the requirement of this milestone. 🥇